src/mcode-pilot/components/TreatmentOptionsSelector/TreatmentOptionsSelector.jsx

```diff
@@ -17,6 +17,11 @@ export default class TreatmentOptionsSelector extends Component {
         this.setState({ expanded: !this.state.expanded });
     };
 
+    shouldComponentUpdate(nextProps, nextState) {
+        // return this.props.subTitle.props.children[0].props.children !== nextProps.subTitle.props.children[0].props.children || nextState.expanded !== this.state.expanded;
```

Comment:
[JA6] Do we still need this commented out line? Will it be used for anything in the future?